app/models/miq_worker.rb

```diff
@@ -182,12 +182,7 @@ def self.settings_name
   end
 
   def self.corresponding_runner
-    @corresponding_runner ||=
-      if const_defined?(:Runner, false)
-        self::Runner.name
-      else
-        settings_name.to_s.camelize
-      end
+    @corresponding_runner ||= self::Runner.name
```

Comment:
We can likely kill the entire method, and simply use `self::Runner` directly. I just didn't want to chase too far up the chain in one go.